arenadata/readme.md

```diff
@@ -163,6 +163,17 @@ bash arenadata/scripts/run_behave_tests.bash
 bash arenadata/scripts/run_behave_tests.bash gpstart gpstop
 ```
 
+## Building with alternative compiler
+
+You can specify a custom compiler to build gpdb with. For that, you need to specify CC and CXX arguments
+with --build-arg option. You can also skip unit tests with SKIP_UNITTESTS flag. For example, building
+with clang without running unit tests requires:
+```bash
+docker build -t gpdb7_regress:latest -f arenadata/Dockerfile . --build-arg CC=clang --build-arg CXX=clang++ --build-arg SKIP_UNITTESTS=true
```

Comment:
could I do so
```sh
docker build -t gpdb7_regress:latest -f arenadata/Dockerfile . --build-arg CC=clang
```
or so
```sh
docker build -t gpdb7_regress:latest -f arenadata/Dockerfile . --build-arg CXX=clang++
```
?

Reply:
That will probably leave the other compiler as `g++` or `gcc` and probably cause compiler personality incompatibilities?

Reply:
You can do whatever you want, these values are here for you so you can provide any compiler, not just clang. Just don't do like in your example if you don't want compiler errors, there's readme file which explain how to use clang

Reply:
It's a strange suggestion and I don't understand why I should do this way

Reply:
question and suggestion are still relevant

Reply:
Explain the advantages of such approach

Reply:
We only have two compilers installed in our docker file and they only work in pairs, so I don't see any point in allowing arbitrary compilers to be specified (which aren't in our docker file anyway), as well as non-working combinations of compilers.